examples/Assertions/Basic/test_plan.py

```diff
@@ -60,7 +60,8 @@ def test_basic_assertions(self, env, result):
         result.true("foo" == "foo", description="Boolean Truthiness check")
         result.false(5 < 2, description="Boolean Falseness check")
 
-        result.fail("This is an explicit failure.")
+        result.passing("This is a passing assertion")
+        result.failing("This is an explicit failure.")
```

Comment:
I'm not a big fan of this rename from fail -> failing. It's not really consistent with the rest of the existing assertions - most other assertions are verbs (equal, match, log etc.) not nouns. I think that result.fail() reads better too. Was there a specific reason you made this change or just personal preference?

Reply:
Another thing, this rename will be a breaking change for any existing users of `result.fail`. Is it really worth breaking user's tests for?